.travis.yml

```diff
@@ -1,5 +1,12 @@
 language: python
+
 python:
-  - 2.7
-install: pip install -q --use-mirrors tox
-script: tox
+  - "3.4"
+  - "2.7"
+  - "pypy"
```

Comment:
actually travis seems to have some permission issue so disabled pypy3 from travis for now but still left in tox config